src/components/avatar/__tests__/AvatarInitials.test.tsx

```diff
@@ -9,9 +9,20 @@ describe('components/avatar/AvatarInitials', () => {
         expect(wrapper.is('span.avatar-initials.test')).toBeTruthy();
     });
 
-    test('should render the initials', () => {
-        const wrapper = shallow(<AvatarInitials name="hello world" />);
-        expect(wrapper.text()).toEqual('HW');
+    test.each`
+        name                               | initials
+        ${'hello'}                         | ${'HH'}
+        ${'hello world'}                   | ${'HW'}
+        ${'hello world (personal)'}        | ${'HW'}
+        ${'hello world (personal acct)'}   | ${'HW'}
+        ${'hello world  (personal acct)'}  | ${'HW'}
+        ${' hello world (personal acct)'}  | ${'HW'}
+        ${'hello  world  (personal acct)'} | ${'HW'}
```

Comment:
Should we also test names with other types of brackets?

Reply:
added a few like that, and ability to handle 1+ pairs good call